omnibus-mainnet.toml

```diff
@@ -127,3 +132,26 @@ target = ["legacyMarket.Proxy"]
 fromCall.func = "owner"
 func = "setPauseMigration"
 args = [true]
+
+# Governance Councils Configuration
+[var.governance]
+governance_package = "synthetix-governance-satellite:3.0.5"
+governance_snapshot_record_package = "synthetix-snapshot-rewards:3.5.0"
+governance_salt = "synthetix-governance"
```

Comment:
salt must equal main `settings.salt`
Do not hardcode salt as it comes with problems down the line (we have this issue with perps): there will be collisions in the future.
```suggestion
```

Remove `governance_salt` settting and replace with `salt` in all the deployments.